src/settings.ts

```diff
@@ -662,6 +662,9 @@ Object.freeze(ALWAYS_EMPTY_ARRAY);
 export const ALWAYS_NOOP_FUNCTION = () => {};
 Object.freeze(ALWAYS_NOOP_FUNCTION);
 
+// NOTE: flag for components that display Not Found screen or use some mitigation when no venueId or no venue exists
+export const ALLOW_NO_VENUE: boolean = true;
```

Comment:
not sure about this flag. I don't see the case so far where we might need a page without a venue.
And as it's related to the NavBar and I don't think we should show NavBar if there is no current venue, my suggestion would be to go without it until we really need that flag.

Reply:
@sunny-viktoryia I'm curious, is navbar without a title an empty navbar? since it still does show the profile image that is clickable and the logo that is also clickable

Reply:
> not sure about this flag. I don't see the case so far where we might need a page without a venue.

- open https://staging.sparkle.space/admin/invalid
- open https://staging.sparkle.space/admin-ng/missing

Reply:
As per https://doesitsparkle.slack.com/archives/C01700P4QCT/p1631548197056100?thread_ts=1631516472.052000&cid=C01700P4QCT

non-empty `NavBar` shown above `NotFound` with the `ALLOW_NO_VENUE` flag removed since enough time has passed from the backport origin where it made sense to use it.

Reply:
flag removed, BM is over, most PRs merged @ using `venueId && hasVenue` check as per slack thread @ d3d4cf50d3102d297ee504e1e6b4762bf039f2fa